### PR TITLE
Accessibility changes

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: 
Issue Type:https://aottech.atlassian.net/browse/FWF-2632
https://aottech.atlassian.net/browse/FWF-2621

# Changes
1.added labels 
2.Fixed contrast errors 

# Screenshots (if applicable)
image
image
image

### CODE REVIEW

File: forms-flow-web/src/components/Form/List.js [hunk 1]
```diff
@@ -608,9 +608,9 @@ const List = React.memo((props) => {
             {/*<img src="/form.svg" width="30" height="30" alt="form" />*/}
             <div className="flex-item-left">
               <div style={{ display: "flex" }}>
-                <h3 className="task-head" style={{ marginTop: "3px" }}>
+                <span className="task-head" style={{ marginTop: "3px" }}>
                   <i className="fa fa-wpforms" aria-hidden="true" />
-                </h3>
+                </span>
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/List.js:613]: Please check the hover colour sometime the span hover will be black

File: forms-flow-web/src/components/Form/List.scss [hunk 2]
```diff
@@ -128,4 +128,10 @@
         flex: 100%;
         justify-content: flex-start;
     }
+}
+.page-item .page-link {
+    color: #0072ed !important;
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/List.scss:133]: Could you please use color variable

**Review by shuhaib-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/List.js (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-web/src/components/Form/List.js:613]: checked , no issues

File: forms-flow-web/src/components/Form/List.scss (hunk 2 above)
Comment (fahad-aot) [reply forms-flow-web/src/components/Form/List.scss:133]: ok

File: forms-flow-web/src/components/ServiceFlow/ServiceFlow.scss [hunk 3]
```diff
@@ -435,7 +435,7 @@ span[dat-title]:hover:after,
       cursor: not-allowed;
       a {
         cursor: not-allowed;
-        color: #d4d4d5;
```
Comment (Josephalexantony-aot) [inline forms-flow-web/src/components/ServiceFlow/ServiceFlow.scss:438]: redundant color property.

**Review by Josephalexantony-aot — APPROVED** [review]

**Pull request merged**